resources/schema/spidacalc/results/clearance_results.schema

```diff
@@ -2,7 +2,36 @@
   "id": "#/spidacalc/results/clearance_results.schema",
   "description": "SPIDACalc clearance results",
   "type": "object",
+  "required": [
+    "id",
+    "status"
+  ],
```

Comment:
unless we aren't serializing them when they're empty, it probably makes sense for results, violations, and itemIds to be required